common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -1296,6 +1296,9 @@ def get_submission_metadata(self, answers, correct_map):
                 'correct': is_correct,
                 'variant': variant,
             }
+            # Add group_label in event data only if the responsetype contains multiple inputtypes
+            if answer_input.response_data.get('group_label'):
+                input_metadata[input_id]['group_label'] = answer_input.response_data.get('group_label')
```

Comment:
@stroilova For multiple input types (as stated in this wiki https://openedx.atlassian.net/wiki/display/A11Y/Custom+Python+Evaluated+Input) we'll have group label for complete group/question and individual label attribute for each input type. Due to this new handling we've added group_label to the event data. I wanted to let you know because analytics team could be interesting in such information in future.